nats-streaming-http-connector/README.md

```diff
@@ -13,12 +13,11 @@ The job of the connector is to read messages from the subject, call an HTTP endp
 - `CONTENT_TYPE`: Content type used while creating post request
 - `NATS_SERVER`: NATs connection string, like `nats://127.0.0.1:4222`
 - `CLUSTER_ID`: StanClusterID to form a connection to the NATS Streaming subsystem
-- `CLIENT_ID`: Which is used by the server to uniquely identify, and restrict, a given client.
```

Comment:
Can you also update the docs link to point to latest KEDA docs on Line number 3?

Reply:
done pointing it to https://keda.sh/docs/2.2/scalers/nats-streaming/ now